### DISCUSSION TITLE
web-worker如何在resize窗口时重置大小自动适配

### DISCUSSION THREAD

**Discussion by humm100** [Tencent/libpag#2043] [Q&A]
使用web-worker时，如何在resize窗口时重置大小自动适配

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

如果你还在使用 libpag 且问题仍然存在，麻烦回复一下当前版本号（4.4 / 4.5 系列）和最小复现 demo，我们会优先跟进；如果不再需要跟进，本条就先关闭了。（stale-close）
